Resources/views/ContentType/view_content_type.html.twig

```diff
@@ -19,130 +19,143 @@
 {% endblock %}
 
 {% block header_title %}
-    <h1 class="ez-page-header-name ez-contenttype-icon ez-contenttype-icon-{{ content_type.identifier }}">{{ content_type_name }}</h1>
+    <h1 class="ez-page-header-name ez-contenttype-icon ez-contenttype-icon-{{ content_type.identifier }}" data-icon="">{{ content_type_name }}</h1>
```

Comment:
here I did that at the beginnig to have the margin but I can move that on ez-contenttype-icon ?